sdk/bolos_syscalls_2.0.h

```diff
@@ -60,6 +60,7 @@
 #define SYSCALL_cx_bn_is_prime_ID_IN 0x6000ef6bUL
 #define SYSCALL_cx_bn_next_prime_ID_IN 0x6000f070UL
 #define SYSCALL_cx_bn_rng_ID_IN 0x6001dd7cUL
+#define SYSCALL_cx_bn_gf2_n_mul_ID_IN 0x05000046UL
```

Comment:
The syscall is not available on the corresponding device versions (Nano S 2.0.0, Nano S 2.1.0, Nano X 2.0.0, Nano X 2.0.2), so the syscall shouldn't be added to be compliant.
The syscall ID is already in `bolos_syscalls_unified_sdk.h`.

Reply:
Fixed in latest push